### DISCUSSION TITLE
не фурычит

### DISCUSSION THREAD

**Discussion by ldslvlao** [bol-van/zapret#812] [Q&A]
короче, у меня последняя версия запрета (1.6.2) и у меня так-же удалён winDivert так вот.
КАК ТОЛЬКО Я ОТКРЫВАЮ ЛЮБОЙ БАТНИК ТО ОН ОТКРЫВАЕТСЯ НА МИЛИСЕКУНДУ И НЕ РАБОТАЕТ ВОЙС я пробовал всё что только нашёл, я хз чё делать если знаете помогите

**Comment by electrifying** [comment]
В репе откуда батники брали и спрашивайте

**Comment by ldslvlao** [comment]
> В репе откуда бантики брали и спрашивайте

что ты несёшь?

**Comment by VirRus77** [comment]
> > В репе откуда бантики брали и спрашивайте
> 
> что ты несёшь?

Версия 1.6.2 не присутствует в текущем репозитории.
Все версии zapret лежат тут: https://github.com/bol-van/zapret/releases
Всё остальное не является результатом работы автора.
Т.е. вас попросили обратиться к автору ВАШЕЙ версии "zapret".
